blog-posts/gatsby-ssr-caching.md

```diff
@@ -0,0 +1,95 @@
+---
+path: "/blog/cache-gatsby-ssr-page-on-gatsby-cloud/"
+date: "2022-02-11"
+title: "Cache Gatsby SSR pages on Gatsby Cloud"
+featuredImage: images/gatsby-ssr-caching.jpg
+author: "Fabian Dietenberger"
+authorSummary: "Senior Developer at Satellytes"
+teaserText: Learn how to cache Gatsby SSR page on Gatsby Cloud
+seoMetaText: Lean how to cache Gatsby SSR pages on Gatsby Cloud to boost page speed. Your users won't see a difference to SSG or DSG pages anymore.
+leadboxText: Want to learn more about creating blazing fast pages? Then join us to learn more!
+attribution:
+    creator: Shiro Hatori
+    source: https://unsplash.com/photos/WR-ifjFy4CI
+---
+
+Since Gatsby 4 you can do [server side rendering](https://www.gatsbyjs.com/docs/how-to/rendering-options/using-server-side-rendering/) 
+of pages. This means, the page gets rendered at runtime, and not at buildtime.
+
+When serving a page via SSR, efficient caching becomes crucial as the browser won’t show anything until the whole page 
+is rendered on the server and delivered to the browser. Caching can be done on multiple levels in your architecture, in 
+this article we will focus on caching at the edge with a CDN.
+
+If you host your Gatsby page with Gatsby Cloud, you have a CDN out of the box. The CDN is powered by Fastly, but that’s 
+just an implementation detail. Caching for your static (SSG) or deferred (DSG) pages is done automatically, so no need 
+to configure anything here. But if you want to do SSR, you need to take care of the correct caching behavior by yourself.
+
+> This post was inspired by the excellent video from [Remix on Youtube](https://www.youtube.com/watch?v=bfLFHp7Sbkg) 
+> If you want to deep dive to CDN caching with SSG and SSR, you should have a look! 
+
+## The Cache Control Header
+
+Caching in Gatsby Cloud can be controlled via the HTTP caching header `Cache-Control`. This header has multiple 
+[directives](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Cache-Control#directives), the most important are:
```

Comment:
I'd (very) slightly prefer a different choice of wording - this sounds a bit too german to my ears. how about "..., most importantly:..."?